portal/src/main/scala/cromwell/pipeline/datastorage/dto/management/DeactivateUserRequestById.scala

```diff
@@ -0,0 +1,12 @@
+package cromwell.pipeline.datastorage.dto.management
+
+import cromwell.pipeline.datastorage.dto.UserId
+import play.api.libs.json.{Json, OFormat}
+
+final case class DeactivateUserRequestById(userId: UserId)
+
+object DeactivateUserRequestById {
+
+  implicit val userIdFormat: OFormat[UserId] = Json.format[UserId]
```

Comment:
Formatter for `UserId` does not belong here. It has nothing to do with `DeactivateUserRequestById` class.
Although this is debatable, since `UserId` is just a wrapper on a string, therefore it should be formatted directly to a `JsString`.